source/_posts/2018-10-20-culture-of-cybersecurity.markdown

```diff
@@ -0,0 +1,43 @@
+
+So how do we do it? We have teams dedicated to security that work directly within engineering. These teams have various responsibilities such as ownership of scanning tools and vulnerability tracking. My team's goal is to bridge the gap by injecting security as a first-class citizen in the software development lifecycle. When working with developers, you have to make the right thing to do the easy thing to do. This is no different when it comes to security. In order to make security easy, we scan, assess, and create a plan for our developers to remediate their vulnerabilities. We promote the tools for scanning, help teams understand the results, and identify fixes for vulnerabilities. We run a monthly cybersecurity meetup which we use as a venue for associates to speak and learn about varying security topics.
+
+<div align="center">
+  <table>
+    <tr>
+      <td align="center">
+        {% img center /assets/2018-10-29-culture-of-cybersecurity/cybersecurity-1.jpg 450px %}
+      </td>
+      <td align="center">
+        {% img center /assets/2018-10-29-culture-of-cybersecurity/cybersecurity-2.jpg 450px %}
+      </td>
+    </tr>
+  </table>
+  <sub>Andy Nelson opening the September edition of the Cybersecurity meetup and Sebastian Brown presenting at the July edition.</sub>
+</div>
+
+We also take advantage of opportunities like Cybersecurity Awareness month. We bridge organizational gaps to host a variety of security focused events, engaging associates in development, security, and operations to facilitate better relationships and collaboration. Events like these lower the barrier to entry for our developers to learn more secure practices, and embrace and celebrate the progress we are making in our security journey. We kicked off the activities this month with an external tech talk from [Britney Hommertzheim](https://twitter.com/bhommertzheim). Britney, the Director of Information Security at [AMC Theatres](https://www.amctheatres.com), presented on how we can better integrate security teams and developers. It was a great talk and you can watch the talk on our [YouTube](https://youtu.be/DTYiNidFjzM) channel. We invited another external speaker for our Cybersecurity meetup a few weeks ago too. Caleb Christopher, a Technical Business Adviser at [Allegiant Technology](https://www.allegianttechnology.com), gave a great talk titled "Defeating Email Fraud with DMARC". Along side those 2 events, we have held lunch and learns throughout the month, and are wrapping it all up with an hour of security focused lightning talks tomorrow.
```

Comment:
Minor, is there a link to reference for Cybsecurity Awareness month? Not sure what the official one is...

https://staysafeonline.org/ncsam/
https://www.dhs.gov/national-cyber-security-awareness-month

Reply:
I think its the second one, I like this idea. I hadn't thought about linking out to it. I added this link: https://github.com/cerner/cerner.github.io/pull/112/commits/00dbdb60f7ac2ea0d7bebe622b28a78027648e21